django/forms/models.py

```diff
@@ -347,7 +347,16 @@ def _update_errors(self, errors):
         # Override any validation error messages defined at the model level
         # with those defined at the form level.
         opts = self._meta
-        for field, messages in errors.error_dict.items():
+
+        # Relates to #24706 - allowing models generated by
```

Comment:
I don't think the comment is needed, it's just standard handling of non-dict ValidationErrors, it could useful outside of the `construct_instance` scenario.